CIP-governance-metadata-cc-rationale/README.md

```diff
@@ -0,0 +1,133 @@
+---
+CIP: ?
+Title: Governance metadata - Constitutional Committee vote rationale
+Category: Metadata
+Status: Proposed
+Authors:
+    - Ryan Williams <ryan.williams@intersectmbo.org>
+    - Eystein Magnus Hansen <eysteinsofus@gmail.com>
+Implementors: []
+Discussions:
+    - https://github.com/cardano-foundation/CIPs/pull/?
+Created: 2024-07-17
+License: CC-BY-4.0
+---
+
+## Abstract
+
+The Conway ledger era ushers in on-chain governance for Cardano via [CIP-1694 | A First Step Towards On-Chain Decentralized Governance](https://github.com/cardano-foundation/CIPs/blob/master/CIP-1694/README.md), with the addition of many new on-chain governance artifacts.
+Some of these artifacts support the linking off-chain metadata, as a way to provide context.
+
+The [CIP-100 | Governance Metadata](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0100) standard provides a base framework for how all off-chain governance metadata should be formed and handled.
+But this is intentionally limited in scope, so that it can be expanded upon by more specific subsequent CIPs.
+
+This proposal aims to provide a specification for off-chain metadata vocabulary that can be used to give context to Constitutional Committee votes.
+
+## Motivation: why is this CIP necessary?
+
+### For GA submitters
+- Provide clarity to submitters - why their proposal is unconstitutional?
+
+### For voters
+- Provide clarity to DReps/ada holders/SPOs
+  - to show that the CC is fair and reasonable
+  - so the CC don't get voted out OR "no confidenced" - this instability would risk the perceived legitimacy of the system
+
+## Specification
+
+We define an initial specification for fields which SHOULD be added to constitutional committee votes.
+
+### Extended Vocabulary
+
+The following properties extend the potential vocabulary of [CIP-100](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0100)'s `body` property.
+
+#### `summary`
+
+- A short text field. Limited to `200` characters.
+- Authors SHOULD use this field to clearly state their stance on the issue.
+- Authors SHOULD use this field to succinctly describe their rationale.
+- Authors SHOULD give a brief overview of the main arguments will support your position.
+- This SHOULD NOT support markdown text styling.
+- Compulsory
```

Comment:
If it is a SHOULD field why do we mention Compulsory multiple times?

Reply:
The field can be made technically compulsory via tooling.
But there is no technical way to enforce the content within the field, so I put them to be SHOULDs.